### ISSUE TITLE
Windows get update command

### ISSUE THREAD

**Issue by MaynardMiner** [MaynardMiner/SWARM#27]
Users are stating they are having issues.

Based on photos, it seems the point in the codebase it is happening, is when the miner goes to close the old version, and start the new. 

I will be testing to reproduce, issues was reported as starting to happen in 2.0.4, but I personally have been unable to reproduce after multiple testing attempts.

**Comment by MaynardMiner** [comment]
The issue was determined to be a connection issue to github for updating.

However, it was found that in 2.0.4:

```
  try {Invoke-WebRequest $URI -OutFile $FileName -UseBasicParsing -ErrorAction Stop}catch {$Failed = $true; $line += "Failed To Contact Github For Download! Must Do So Manually"}
```

There was no -Timeout specified. So if connection failed- It would lock up SWARM indefinitely. I caught while designing 2.0.5, and applied fix. 2.0.5 to 2.0.6 should not have issues, and should throw an error that it failed to download miner, so you can attempt again.

**Comment by MaynardMiner** [comment]
Proposed fix applied: 2.0.6
